tests/hardware/plugins/temperature/test_senecaK107.py

```diff
@@ -87,12 +87,12 @@ def test_parse_data(dev: SenecaK107, data: bytes) -> None:
     expected = [
         19.946250000000006,
         20.085000000000008,
-        631.4406250000001,
-        631.4175,
+        float("nan"),
+        float("nan"),
         20.14281249999999,
-        631.4406250000001,
+        float("nan"),
         19.946250000000006,
-        631.4406250000001,
+        float("nan"),
```

Comment:
Might also want to test to see if `MIN_TEMP` is respected here.

Reply:
Yeah this is definitely something we should write tests for, but I imagine it'll require having examples of bytestring data containing temperatures below the minimum limit. I don't know enough about bytestrings to fabricate test data like this so might be worth making a separate issue.